crates/cli/tests/expects_transitive/Direct.roc

```diff
@@ -0,0 +1,14 @@
+interface Direct
+    exposes [
+        addAndStringify,
+    ]
+    imports [
+        Transitive,
+    ]
+
+addAndStringify = \num1, num2 ->
+    Num.toStr (Transitive.add num1 num2)
+
+expect addAndStringify 1 2 == "3"
+
+expect addAndStringify 3 4 == "7"
```

Comment:
Someone more experienced should give advice how relevant it is to test for both `expect` and `expect-fx`.

When in doubt, personally I'd like to test for both -- just to be safe.

Reply:
`expect-fx` is not completely implemented yet, so I think we should leave it out for now.